src/scripts/modules/media/body/embeddables/modals/sims/sims-template.html

```diff
@@ -0,0 +1,13 @@
+<div id="sims-modal" class="modal fade">
+  <div class="modal-dialog">
+    <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+          <h4 class="modal-title">{{simTitle}}</h4>
+        </div>
+        <div class="modal-body">
+            <iframe src="{{simUrl}}" width="960px" height="785px" frameborder='0'></iframe>
```

Comment:
Oh, do all sims have the same height? I thought Patrick had a fixed width but the heights changed depending on the sim.